sng_freepbx_debian_install.sh

```diff
@@ -62,6 +112,10 @@ while [[ $# -gt 0 ]]; do
 			noioncube=true
 			shift # past argument
 			;;
+                --skipversion)
+                        skipversion=ture
```

Comment:
Unsure if this is on purpose but some parts of the script use `var=true` but this line is `var=ture`

FYI @spurnasai @kguptasangoma